app/src/main/java/chat/rocket/android/helper/ServerHelper.java

```diff
@@ -0,0 +1,103 @@
+package chat.rocket.android.helper;
+
+import android.support.annotation.NonNull;
+
+import org.json.JSONObject;
+
+import java.io.IOException;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+public class ServerHelper {
```

Comment:
The name of ServerHelper is vague.
ServerValidationChecker or another name (with the class responsibility) would be better.